components/build/README.md

```diff
@@ -0,0 +1,63 @@
+# `build.rs` utility for Exonum
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/blob/master/LICENSE)
+![rust 1.36.0+ required](https://img.shields.io/badge/rust-1.36.0+-blue.svg?label=Required%20Rust)
+
+This crate simplifies writing `build.rs` for Exonum and Exonum services.
+
+Since `protobuf` is the Exonum default serialization format, `build.rs` is
+mostly used to compile `protobuf` files and generate corresponding code.
+Generated code is used later by the Exonum core and services.
+
+There are three predefined sets of protobuf sources available for use:
```

Comment:
Nit: I'd not mention the number of sets, since it's easy to get wrong (like now, there are four sets mentioned, not three).